VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+            <date>2020-11-14T14:52:12.552+02:00</date>
+         </publicationStmt>
+         
+         <sourceDesc>
+            <p>
+               <cit>
+                  <quote xml:lang="it">
+                     Salterio, seguito dai soliti cantici e dal Weddâse Mâryâm.
+                     16 x 11,6, f. 176 di cui 2 in principio e uno bianco alla fine. Senza data.
+                     Rilegatura in legno non ricoperto. Riposto in una custodia.
+                  </quote>
+                  <bibl>
+                     <ptr target="bm:Zanutto1932Bibliografia"/>
+                     <citedRange unit="page">93</citedRange>
+                     <citedRange unit="item">8</citedRange>
+                  </bibl>
+               </cit>
+            </p>
+         </sourceDesc>
+         
+         <sourceDesc>
+            <msDesc xml:id="ms">
+          
+               <msIdentifier>
+                  <repository ref="INS0453Bibl_Giov_Veroli"/>
+                  <collection/>
+                  <idno>BGV-008</idno>
+                  <altIdentifier>
+                     <idno>Zanutto 8</idno>
+                  </altIdentifier>
+               </msIdentifier>
+          
+               <msContents>
+                  <summary/>
+            
+                   <msItem xml:id="ms_i1">
+                      <locus from="3r" to="175vb"/>
+                     <title type="complete" ref="LIT2701Dawit"/>
+                      <msItem xml:id="ms_i1.1">
+                         <locus target="#3r"/>
+                         <title type="incomplete" ref="LIT4009Naahab"/>
+                         <note>See the transcription in <ref target="#LIT4009Naahab"/>.</note>
+                      </msItem>
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="3r" to="136r"/>
+                        <title type="complete" ref="LIT2000Mazmur"/>
+                        <incipit xml:lang="gez">
+                           <locus target="#3ra"/>
+                           <lb/><hi rend="rubric">ፍካሬ፡ ዘጻድቃን፡ ወዘኃጥአን።</hi>
+                           <lb/>መዝሙር፡ ዘዳዊት። <hi rend="rubric">ሃሌ፡ ሉያ።
+                              <lb/>፩ ብፁዕ፡ ብእሲ፡ ዘኢሖረ፡ በምክረ፡ ረሲአን።</hi> 
+                           <lb/>ወዘኢቆመ፡ ውስተ፡ ፍኖተ፡ ኃጥአን።
+                        </incipit>
+                        <explicit xml:lang="gez">
+                           <locus target="#136r"/>
+                           <lb/>ወረገመኒ፡ በአማልክቲሁ።
+                           <lb/>ወአንሰ፡ ነሣእኩ፡ ዘእምላዕሌሁ፡ ሰይፈ።
+                           <lb/>ወመተርኩ፡ ርእሶ።
+                           <lb/>ወአሰሰልኩ፡ ጽእለተ፡ እምደቂቀ፡ ፳ኤል።
```

Comment:
I wrote ፳ኤል፡ both because in fact it is not a real numeral, but one single word, and because one might be interesting to filter precisely all occurrences of these abbreviated form.